### PR TITLE
Fixed some flow issues

### PR DESCRIPTION
**Pull request by ghost**
Remind me to make a Mac support thingy that’s like the Windows one, but for Max.

### CODE REVIEW

**Comment by ghost** [comment]
Mac*

**Review by nacrt — CHANGES_REQUESTED** [review]
17: Java is not needed at all

**Comment by ghost** [comment]
mf when I use the universal installer it won’t load without java

**Comment by nacrt** [comment]
> mf when I use the universal installer it won’t load without java

it will work on any java version above 8, so installing java isn't required, which is all the end user needs to know

File: thread.txt [hunk 1]
```diff
@@ -12,9 +12,9 @@ and have a lot of convenience for the end user.
 [ATTACH type="full" alt="scthreaddivider.png"]2965072[/ATTACH]
 
 [COLOR=rgb(184, 49, 47)][SIZE=5][U][B]One-click Forge Installation[/B][/U][/SIZE][/COLOR]
-[COLOR=rgb(0, 0, 0)][SIZE=4]This one basically explains itself.
+[COLOR=rgb(0, 0, 0)][SIZE=4]This is self-explanitory
```
Comment (KTibow) [inline thread.txt:15]:
```suggestion
[COLOR=rgb(0, 0, 0)][SIZE=4]Pretty much self-explanatory
```

File: thread.txt [hunk 2]
```diff
@@ -12,9 +12,9 @@ and have a lot of convenience for the end user.
 [ATTACH type="full" alt="scthreaddivider.png"]2965072[/ATTACH]
 
 [COLOR=rgb(184, 49, 47)][SIZE=5][U][B]One-click Forge Installation[/B][/U][/SIZE][/COLOR]
-[COLOR=rgb(0, 0, 0)][SIZE=4]This one basically explains itself.
+[COLOR=rgb(0, 0, 0)][SIZE=4]This is self-explanitory
 Just choose your mods and packs, and click "Install SkyClient" to install.
-You also won't need to install Java, as Minecraft comes with its own version by default.[/SIZE][/COLOR]
+If you're using the Windows installer, you also won't need to install Java, as Minecraft comes with its own version by default.[/SIZE][/COLOR]
```
Comment (KTibow) [inline thread.txt:17]:
```suggestion
You also probably won't need to install Java, as Minecraft comes with its own version by default.[/SIZE][/COLOR]
```

File: thread.txt [hunk 3]
```diff
@@ -61,9 +61,7 @@ MobPacket, ActuallyPacket, Dungeons Texturepack, Compliment, ActuallyMobs, SkyMo
 
 [COLOR=rgb(184, 49, 47)][SIZE=5][U][B]Update your mods and packs with very minimal effort[/B][/U][/SIZE][/COLOR]
 We know how annoying it is to check your mods and packs for updates.
-Simply click the "Update" button, and it will find your mods and packs that are outdated and update them for you!
-We also have a mod called "SkyClient Updater" that installs all the mods you have outdated as soon as you start up.
-[B][SIZE=3]DISCLAIMER: THE UPDATE FUNCTION ON THE INSTALLER DOES NOT WORK DUE TO IT BEING BROKEN. IT SHOULD BE FIXED IN A LATER UPDATE.[/SIZE][/B]
+We have a mod to fix this nusciance called "SkyClient Updater" that installs all the mods you have outdated as soon as you start up.
```
Comment (KTibow) [inline thread.txt:64]:
```suggestion
We have a mod to fix this, SkyClient Updater. It automatically detects what mods you have, and if SkyClient has a newer version, it offers to download it for you.
```

File: thread.txt [hunk 4]
```diff
@@ -73,7 +71,7 @@ You can also access the mods folder quickly by clicking a button in the "Advance
 
 [COLOR=rgb(184, 49, 47)][U][SIZE=5][B]Understand what your mods do and where they come from[/B][/SIZE][/U][/COLOR]
 Many people simply do not know what the mods they have installed actually do, and download them just because a YouTuber recommended them.
-That's why we've added a little guide for every single mod we include in our installer to understand if you want the features in the mod or not.
+That's why we've added small guides for each mod we offer in our installer to understand if you want the features in the mod or not.
 Along with that, you can easily access the Github page, website, video, or whatever source the mod is from, by simply pressing the buttons next to it.
```
Comment (KTibow) [inline thread.txt:75]:
```suggestion
That's why we've added links (like to the mod author's page or to the source code) and guides for every single thing in Skyclient, 
```

**Review by KTibow — CHANGES_REQUESTED** [review]
fix spelling and stuff idk up to you

File: thread.txt (hunk 4 above)
Comment (ghost) [reply thread.txt:75]: I WILL LITERALLY SHIT YOURSELF

WHEN YOU INSTALL MINECRAFT ON MACOS AND LINUX, IT DOES NOT INSTALL JAVA. THATS SOME WEIRD CRAZY SHIT MICROSOFT DOES. ON MACOS YOU MUST SEPRATELY INSTAL JAVA

Comment (nacrt) [reply thread.txt:75]: no it works out of the box

Comment (KTibow) [reply thread.txt:75]: Java is installed by default on macos/linux. Besides the web installer exists.

**Comment by ghost** [comment]
DO YOU NEED ME TO MAKE A DEMONSTRATION ON HOW MACS DIBTBCOME WUTH JAVA (OR AT LEAST ABLE TO RUN THIS)

**Comment by nacrt** [comment]
> DO YOU NEED ME TO MAKE A DEMONSTRATION ON HOW MACS DIBTBCOME WUTH JAVA (OR AT LEAST ABLE TO RUN THIS)

please do that

**Pull request closed without merging**